woohoo.rb

```diff
@@ -0,0 +1,5 @@
+[:a, :b].find { |_| true }
+[:a, :b].detect { |_| true }
+
+puts "SUP!"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.